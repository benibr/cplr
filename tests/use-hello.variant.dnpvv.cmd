../bin/cplr 'puts("Hello.")' -dnpvv
