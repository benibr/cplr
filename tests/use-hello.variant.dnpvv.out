Preparation phase
Generation phase
Generating code
Generating labeled statements
================================================================================
     1	/* main */
     2	int main(int argc, char **argv) {
     3		int ret = 0;
     4	/* statements */
     5	 s0:	puts("Hello.");
     6	/* done */
     7		return ret;
     8	}
================================================================================
Generated: 188 bytes code, 123 bytes dump
Compilation phase
Compiler condition:
  statements_0:1: warning: implicit declaration of function 'puts'
Compilation succeeded
